content/en/tracing/guide/monitor-kafka-queues.md

```diff
@@ -0,0 +1,121 @@
+| `messaging.kafka.consumer_group` | `string` |  Name of the Kafka Consumer Group that is handling the message. Only applies to consumers, not producers.
+| `messaging.kafka.client_id`      | `string` |  Client ID for the Consumer or Producer that is handling the message.                                               |
+| `messaging.kafka.partition`      | `string` |  Partition the message is sent to.                                                                                  |
+| `messaging.kafka.tombstone`      | `string` |  A Boolean that is true if the message is a tombstone.                                                              |
+| `messaging.kafka.client_id`      | `string` |  Client ID for the Consumer or Producer that is handling the message.                                               |
+
+## Special use cases
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+The Datadog Kafka integration works with Kafka version 0.11+, which supports the Header API. This API is used to inject and extract trace context. If you run a mixed version environment, the Kafka broker can incorrectly report the newer version of Kafka. This causes an issue when the tracer tries to inject headers that are not supported by the local producer. Additionally, older consumers are unable to consume the message because of the presence of headers. To prevent these issues, if you run a mixed version Kafka environment with versions older than 0.11, disable context propagation with the environment variable `DD_KAFKA_CLIENT_PROPAGATION_ENABLED=false`.
+
+{{% /tab %}}
+
+{{% tab ".NET" %}}
+
+The [Kafka .NET Client documentation][1] states that a typical Kafka consumer application is centered around a consume loop, which repeatedly calls the Consume method to retrieve records one-by-one. The `Consume` method polls the system for messages. Thus, by default, the consumer span is created when a message is returned and closed before consuming the next message. The span duration is then representative of the computation between one message consumption and the next.
+
+When a message is not processed completely before consuming the next one, or when multiple messages are consumed at once, you can set `DD_TRACE_KAFKA_CREATE_CONSUMER_SCOPE_ENABLED` to `false` in your consuming application. When this setting is `false`, the consumer span is created and immediately closed. If you have child spans to trace, follow [the headers extraction and injection documentation for .NET custom instrumentation][2] to extract the trace context.
+
+The .NET tracer allows tracing Confluent.Kafka since [v1.27.0][3]. The trace context propagation api, if you need it, is available since [v2.7.0][4]
```

Comment:
```suggestion
The .NET tracer allows tracing Confluent.Kafka since [v1.27.0][3]. The trace context propagation API is available since [v2.7.0][4]
```